locales/client/en.json

```diff
@@ -1353,7 +1353,7 @@
   "V2_Form_LikerId_label": "Liker ID",
   "V2_Form_LikerId_placeholder": "Create your Liker ID",
   "V2_Form_LikerId_error_duplicated": "This Liker ID has been registered",
-  "V2_Form_LikerId_error_invalid": "This Liker ID has invalid character",
+  "V2_Form_LikerId_error_invalid": "This Liker ID has invalid character (no uppercase letters or punctuation)",
```

Comment:
```suggestion
  "V2_Form_LikerId_error_invalid": "This Liker ID has invalid character (only lower case alphanumerics are allowed)",
```

Reply:
[🚸 Add validation rules for Liker ID](https://github.com/likecoin/like-co/pull/1772/commits/e2489269b0b72e378870e49632ab3177c7b35476)